pyHalo/Halos/HaloModels/gaussian.py

```diff
@@ -0,0 +1,51 @@
+from pyHalo.Halos.halo_base import Halo
+
+class Gaussian(Halo):
```

Comment:
can you add a test function for this class? in particular, something that illustrates the mass definition